docs/getting-started/cli-oss.md

```diff
@@ -0,0 +1,96 @@
+---
+slug: cli
+title: CLI-only workflow (OSS)
+hide_title: true
+description: Learn how to set up Semgrep, scan your first project for security issues, and view your findings in the CLI.
+tags:
+  - quickstart
+  - cli
+---
+
+import MoreHelp from "/src/components/MoreHelp";
+import Install from "/src/components/procedure/_install-cli.mdx";
+import Login from "/src/components/procedure/_login-activate.mdx";
+
+# CLI-only workflow with Semgrep's OSS Engine
+
+Learn how to set up Semgrep, scan your first project for security issues, and view your findings in the CLI.
+
+## Prerequisites
+
+See [Prerequisites](/prerequisites/) to ensure that your machine meets Semgrep's requirements.
+
+## Set up Semgrep
+
+<Install />
+
+## Scan your project
+
+# Semgrep OSS
+
+Navigate to the root of your repository, and run your first scan. The specific command you use depends on how you want to view the results.
+
+To view the results in the CLI:
+
+```console
+semgrep scan
```

Comment:
If you don't specify a config I'm pretty sure this does nothing?

Reply:
It does (IDK if this is unintended behavior?)

<img width="627" alt="image" src="https://github.com/semgrep/semgrep-docs/assets/16521651/14a52ee3-c2fa-4d17-9104-213a64eb2b2d">

Reply:
This is new!! (Previously it would tell you to specify a config, and recommend config auto). Does the scan output say with what config it's scanning with?

Reply:
It does not, but the results are identical to `semgrep scan --config auto`

Reply:
:| ...we used to tell the user what rulesets are running regardless of log level. i'll make a ticket for the UX. thanks for looking into this.